### PR TITLE
Added Support for Transactional Emails

### PR DESCRIPTION
**Pull request by tylerlutz**
I needed to use the transactional email API to send single emails. Hubspot Requires the Transactional Email Add-On before this can be used. This is the URL to the Hubspot API: https://developers.hubspot.com/docs/methods/email/transactional_email/single-send-overview

### CODE REVIEW

**Comment by MattMSumner** [comment]
@tylerlutz thanks for opening this PR! Sorry it's taken a while for us to get to it.

Do you happen to know if there's any way to test this endpoint? It looks like transaction emails are a special feature that needs to be turned on.

**Comment by tylerlutz** [comment]
You have to have the Transactional Email API enabled on the Hubspot account. We are running a production instance off of this change and it is working perfectly. Let me know if there is anything else I can provide to you.

Thanks,

Tyler Lutz
Software Engineer | tylerlutz.com
270.317.4935

File: lib/emails.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    sendTransactionalEmail (data, cb) {
+        return this.client._request({
+            method: 'POST',
+            path: '/email/public/v1/singleEmail/send',
+            body: data
+        }, cb)
+    }
+}
+
+module.exports = Emails;
```
Comment (MattMSumner) [inline lib/emails.js:14]: Would you mind adding a newline here? Here's a blog post describing why GitHub shows missing newlines at the end of files: https://robots.thoughtbot.com/no-newline-at-end-of-file

**Review by MattMSumner — COMMENTED** [review]
@tylerlutz sweet, would you mind removing the `.DS_Strore` file and the changes in `package-lock.json`? Once that's done I'm comfortable merging this. Thanks again!

**Comment by MattMSumner** [comment]
@tylerlutz I went ahead and cherry picked your commit and opened a new PR to get this merged.

Thanks again, closing in favor of https://github.com/MadKudu/node-hubspot/pull/136

**Pull request closed without merging**